examples/chemistry/qchem_modelling_basics.ipynb

```diff
@@ -476,7 +486,7 @@
    "source": [
     "## Advanced Usage of PySCF<a class=\"anchor\" id=\"5\"></a>\n",
     "\n",
-    "Tangelo currently relies on [PySCF](https://pyscf.org/). Advanced usage of this open-source quantum chemistry package enables the use of many analysis tools. We show a couple of examples where advanced users can extract more information from a PySCF calculation.\n",
+    "[PySCF](https://pyscf.org/) is one of the classical chemistry backend supported by Tangelo (Psi4 is also supported). Advanced usage of this open-source quantum chemistry package enables the use of many analysis tools. We show a couple of examples where advanced users can extract more information from a PySCF calculation.\n",
```

Comment:
backendS. I forgot to add a link to psi4, can you insert a link to the main page of the project ?

Reply:
Oh good catch